utils/utils.js

```diff
@@ -249,15 +249,16 @@ var exportProductsToCsvFileFormat = function(commandName, products, outlets){
   var fs = require('fs');
 
   // NOTE: could also use _.filter() to get keys whose immediate value isn't an object or array
-  var headers = ['id','handle','sku',
+  var headers = ['id','name','handle','sku',
```

Comment:
I'm afraid this will break the default format that the code which used this previously expected. Please pass the headers you want into this method as the last argument and use them that way.

`var exportProductsToCsvFileFormat = function(commandName, products, outlets, customCsvHeaders){`

`var headers = customCsvHeaders || ['id','handle','sku',`